pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -863,6 +875,17 @@ def _initialize(self):
         # report that the source spoke has been initialized
         self.initialize_done()
 
+    def _enable_autodetect_with_device(self, device):
+        device_label = getattr(device.format, "label", "") or ""
+        self._enable_autodetect(device.name, device_label)
+
+    def _enable_autodetect(self, device_name, device_label):
```

Comment:
This method is more about showing autodetected devices than about enabling the autodetection. Could you change the name, please?